Scripts/script-ReadFile.yml

```
@@ -10,7 +10,14 @@ script: |-
       maxFileSize = 1024**2
   res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
   with open(res[0]['Contents']['path'], 'r') as f:
-      data = f.read(maxFileSize).decode('unicode_escape').encode('utf-8')
+      data = f.read(maxFileSize)
+      try:
+          data = data.decode('unicode_escape')
```

I am not clear about why we are doing decode with 'unicode_escape'. Most files we will want to read will be encoded in utf-8 and not unicode_escape. 
```suggestion
          data = data.decode(demisto.args().get("encoding"), 'replace')
```